### PR TITLE
Add footer to admin views

### PR DESCRIPTION
**Pull request by jessieay**
* Also re-add font-awesome css
  I removed it a while ago thinking it was for fonts, but it was for
  icons like the github icon
* Closes https://github.com/18F/micropurchase/issues/1270

### CODE REVIEW

File: app/views/layouts/admin.html.erb [hunk 1]
```diff
@@ -3,23 +3,26 @@
 <head>
   <meta charset="utf-8">
   <meta name="ROBOTS" content="NOODP" />
-  <%= stylesheet_link_tag    'application', media: 'all', debug: true %>
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css">
```
Comment (harrisj) [inline app/views/layouts/admin.html.erb]: Not really blocking this, but I wonder if we have any policy about using third-party CDNs or not

File: config/initializers/assets.rb [hunk 2]
```diff
@@ -1,3 +1,3 @@
 Rails.application.config.assets.version = (ENV["ASSETS_VERSION"] || "1.0")
 Rails.application.config.assets.precompile += %w( jquery.js )
-Rails.application.config.assets.precompile += %w( uswds/us_flag_small )
+Rails.application.config.assets.precompile += %w( uswds/us_flag_small.png )
```
Comment (harrisj) [inline config/initializers/assets.rb]: Can you fix this Rubocop issue?

Comment (harrisj) [inline config/initializers/assets.rb]: Eh, never mind

File: app/views/layouts/admin.html.erb (hunk 1 above)
Comment (jessieay) [inline app/views/layouts/admin.html.erb]: goooood question! we are using a jquery cdn too! (google)

**Pull request merged**